### PR TITLE
updating TkMap script

### PR DESCRIPTION
**Pull request by fioriNTU**
Update the TkMap script for 2017 and add a legend to the quality test Tracker Map.

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh [hunk 1]
```diff
@@ -188,22 +198,22 @@ do
...
 
-    #cmsRun ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py print globalTag=${GLOBALTAG} runNumber=${Run_numb} dqmFile=${file_path}/$dqmFileName  # update GlobalTag
     cmsRun ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py print globalTag=${GLOBALTAG} runNumber=${Run_numb} dqmFile=${file_path}/$dqmFileName  detIdInfoFile=${detIdInfoFileName} # update GlobalTag
 
 # rename bad module list file
 
      mv QTBadModules.log QualityTest_run${Run_numb}.txt
 
+# put color legend in the TrackerMap     
+    /usr/bin/python ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/scripts/LegendToQT.py QTestAlarm.png FinalLegendTrans.png
```
Comment (davidlange6) [inline DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh]: why should you use the system python here? @fioriNTU

**Comment by davidlange6** [comment]
otherwise, @fioriNTU - was the png added on purpose? We don't usually have those in a source repository

File: DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh (hunk 1 above)
Comment (fioriNTU) [inline DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh]: @davidlange6 I assume this has been tested in a different machine where the package PIL for python has been installed manually. Is there any problem in using the system python?

**Comment by fioriNTU** [comment]
@davidlange6 for the png, I tryed to think to s different place where to store the image, but actually the best would be in the repository itself. Is it forbidden?

**Comment by davidlange6** [comment]
> On Jun 1, 2017, at 8:12 AM, fioriNTU <notifications@github.com> wrote:
> 
> @fioriNTU commented on this pull request.
> 
> In DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh:
> 
> >      cmsRun ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py print globalTag=${GLOBALTAG} runNumber=${Run_numb} dqmFile=${file_path}/$dqmFileName  detIdInfoFile=${detIdInfoFileName} # update GlobalTag
>  
>  # rename bad module list file
>  
>       mv QTBadModules.log QualityTest_run${Run_numb}.txt
>  
> +# put color legend in the TrackerMap     
> +    /usr/bin/python ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/scripts/LegendToQT.py QTestAlarm.png FinalLegendTrans.png
> 
> @davidlange6 I assume this has been tested in a different machine where the package PIL for python has been installed manually. Is there any problem in using the system python?

yes, this is likely to cause problems - 

I guess I haven't followed what this PR is meant to do. It seems you are adding a legend to a root plot by taking a png and overlaying it with another png. Is that right? Can you not just add a TLegend capability to TrackerMap itself?

**Pull request closed without merging**